netty/src/main/java/io/grpc/transport/netty/ProtocolNegotiators.java

```diff
@@ -86,49 +87,59 @@ public static ProtocolNegotiator tls(final SslContext sslContext,
     Preconditions.checkNotNull(sslContext, "sslContext");
     Preconditions.checkNotNull(inetAddress, "inetAddress");
 
-    final ChannelHandler sslBootstrapHandler = new ChannelHandlerAdapter() {
```

Comment:
FYI, you could have left the class defined within the method and just given it a name (so no longer anonymous). I know it is rare to see non-anonymous classes defined in methods though.